pypiprivate/config.py

```diff
@@ -1,6 +1,9 @@
 import os
-from ConfigParser import SafeConfigParser
 
+try:
+    from ConfigParser import SafeConfigParser
+except ModuleNotFoundError:
```

Comment:
I think you should catch `ImportError` here because `ModuleNotFoundError` is newly introduced in 3.6 and this tool supports earlier versions of Python as well. `ModuleNotFoundError` is a subclass of `ImportError` so it should work fine for both Python 2 and 3.